static/js/project/controllers/project.controller.js

```diff
@@ -58,8 +58,19 @@
       };
 
       self.myProjects = [];
+      self.myModules = [];
       Project.getRequesterProjects().then(function(data) {
         self.myProjects = data[0];
```

Comment:
this should be sufficient, looping in the templates will give you everything you need